### ISSUE TITLE
make `datalad-recursiveinstall skip` default for sourcedata

### ISSUE THREAD

**Issue by pvavra** [psychoinformatics-de/datalad-hirni#157]
Going through the datalad handbook, I noticed [a footnote](http://handbook.datalad.org/en/latest/basics/101-116-sharelocal.html#f1) mentions the option to make sub-datasets to not be installed recursively, unless explicitly requested. I think this should be set to be the default for `sourcedata` when converting a dataset to bids. 

Installing the bids dataset later as input would most likely not need access to the sourcedata along with it.

The simplest solution would be to have the docs mention that option and show how to set it manually.

If possible, automatically setting that option would be preferable. However, I'm not sure which step should be doing that..

**Comment by bpoldrack** [comment]
I agree that this config would be good to have in the BIDS dataset whenever it was built the way intended by hirni. However, this

> I'm not sure which step should be doing that..

is the problem indeed. The `cfg_bids` procedure (called by `datalad create -c bids mybids_dataset` for example) could do that. But: Nobody is required to use hirni to get such a `sourcedata`. And even if you follow hirni, you are not required to install the raw dataset into `sourcedata` directly. You could install it somewhere underneath `sourcedata` and still be BIDS compliant. Leading to: There's no hirni or neuroimaging related command or procedure that at the time of it's execution knows for sure what that configuration should look like. Only whoever installs that dataset into the BIDS dataset knows.

So, adding that to the docs - yes. Setting it automatically - I can't see where.
